src/components/Landing/Payments.tsx

```diff
@@ -44,20 +60,14 @@ export default function Payments() {
                 borderBottom: '1px solid ' + colors.stroke.tertiary,
               }}
             >
+              <ProjectHandle project={e.project} />
```

Comment:
minor style thing—don't we want this inside the flex div?

Reply:
ohhh damn good catch. Yes i think we do. brb